File: CodCad/ProgramacaoBasica/OrdenacaoSimples.cpp
    #include <iostream>
    #include <algorithm>
     
    using namespace std;
     
    int vetor[100000];
     
    int main () {
    	int n = 0, i = 0;
     
    	cin >> n;
     
    	for(i = 0; i < n; i++) {
    		cin >> vetor[i];	
    	}
     
    	sort(vetor, vetor+n);
    	n--;
     
    	for(i = 0; i < n; i++) {
    		cout << vetor[i] << " ";
    	}
     
    	cout << vetor[n] << endl;
     
    	return 0;
    }

File: ProgramacaoIntermediaria/OrdenacaoSimples.py
n = int(input())
vetor = input().split()

for i in range(len(vetor)):
	vetor[i] = int(vetor[i])

def bubbleSort(array):
    for i in range(len(array)):
        for j in range(len(array)- 1):
            if int(array[j]) > int(array[j+1]):
                aux = array[j+1]
                array[j+1] = array[j]
                array[j] = aux
    return array

vetor.sort()

for i in vetor:
    print(i, end=" ")